src/apps/explorer/components/OrderWidget/OrderDetails.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons'
+
+import { RawOrder } from 'api/operator'
+
+import { SimpleTable } from 'components/common/SimpleTable'
+
+const COLUMN_COLOR = '#252535'
+
+const Table = styled(SimpleTable)`
+  > tbody > tr {
+    grid-template-columns: minmax(10rem, 25rem) auto;
+
+    background: ${({ theme }): string => theme.bg2};
+    border-right: 0.1em solid;
+    border-color: ${COLUMN_COLOR};
+
+    :first-of-type {
+      /* border on top of the first row to close the table */
+      border-top: 0.1rem solid ${COLUMN_COLOR};
+    }
+
+    &:last-of-type {
+      /* Remove weird block at the end of the table */
+      margin: 0;
+    }
+
+    > td {
+      justify-content: flex-start;
+
+      :first-of-type {
+        /* First column same color as border */
+        background: ${COLUMN_COLOR};
+
+        /* Question mark */
+        > svg {
+          margin-right: 2rem;
+        }
+
+        /* Column after text on first column */
+        ::after {
+          content: ':';
+        }
+      }
+
+      height: 5rem;
+
+      :not(:first-of-type) {
+        padding-left: 2.5rem;
+      }
+    }
+  }
+`
+
+// TODO: either use a RichOrder object or transform it here
+// TODO: for that we'll need token info (decimals, symbol)
+export type Props = { order: RawOrder }
+
+export function OrderDetails(props: Props): JSX.Element {
+  const { order } = props
+  const { uid } = order
+
+  const questionMark = useMemo(() => <FontAwesomeIcon icon={faQuestionCircle} />, [])
+
+  return (
+    <Table
+      body={
+        <>
```

Comment:
shouldnt this just be `tbody`, why is it a fragment?

Reply:
Because this is the `<SimpleTable>` component.
Remember you suggested to take `body` as a prop instead of children?

Reply:
yeah ignore what i said. could also just expect <tbody> but this is also fine